### ISSUE TITLE
Error: Failed to load font file, Unknown font format when using next/font/local

### ISSUE THREAD

**Issue by milkyfuckinway** [vercel/next.js#74071]
### Link to the code that reproduces this issue

https://github.com/milkyfuckinway/next-error-unknown-font-format

### To Reproduce

1. Start the application "npm run dev"
2. Errors "Failed to load font file: (path) Unknown font format" will appear in console

### Current vs. Expected behavior

Expect no errors

### Provide environment information

```bash
Operating System:
  Platform: win32
  Arch: x64
  Version: Windows 10 Enterprise
  Available memory (MB): 16244
  Available CPU cores: 12
Binaries:
  Node: 23.4.0
  npm: 10.9.2
  Yarn: N/A
  pnpm: N/A
Relevant Packages:
  next: 15.1.1 // Latest available version is detected (15.1.1).
  eslint-config-next: N/A
  react: 19.0.0
  react-dom: 19.0.0
  typescript: 5.7.2
Next.js Config:
  output: N/A
```

### Which area(s) are affected? (Select all that apply)

Font (next/font)

### Which stage(s) are affected? (Select all that apply)

next dev (local), next build (local)

**Comment by milkyfuckinway** [comment]
Fixed with downgrading node from 23 to 22

**Comment by LinusDierheimer** [comment]
Is there a fix for people using node 23?

**Comment by wesbos** [comment]
this should be opened - having this issue as well

**Comment by milkyfuckinway** [comment]
> this should be opened - having this issue as well

Done, if tou can please add reproduction link, ive deleted mine, sorry

**Comment by wesbos** [comment]
looks to be upstream in fontkit: https://github.com/foliojs/fontkit/issues/356

**Comment by jamtycle** [comment]
I am having this issue with bun 1.1.42 and node 23.4.0. 
NextJS version: 15.1.2

Trying to load the font using:
```js
import localFont from "next/font/local"; 

export const outfit = localFont({
    src: "../public/fonts/outfit.ttf",
    preload: true,
    variable: "--font-outfit",
});
```
results in: 
> `Failed to load font file: .../public/fonts/outfit.ttf 
> Error: Unknown font format`

Also tried with `woff2` files, same error.
Using fonts from `"next/gont/google"` seems to work.

**Comment by xyt-dev** [comment]
This works normally until Node 23.4.0:

const vivo_sans = localFont({
  src: [
    { path: "Font/vivoSans-Regular.ttf", weight: '500' },
    { path: "Font/vivoSans-Bold.ttf", weight: '700' },
  ],
  variable: '--font-vivo-sans',
})

Failed to load font file: Font/vivoSans-Bold.ttf
Error: Unknown font format
 ○ Compiling / ...
The user aborted a request.

Retrying 1/3...
The user aborted a request.

Retrying 1/3...
The user aborted a request.

Retrying 1/3...

<--- Last few GCs --->

[506930:0x5f63b7b4e000]   208231 ms: Scavenge (interleaved) 6435.6 (6444.3) -> 6432.7 (6452.8) MB, pooled: 0 MB, 7.87 / 0.00 ms  (average mu = 0.229, current mu = 0.186) allocation failure;
[506930:0x5f63b7b4e000]   209436 ms: Mark-Compact (reduce) 6433.0 (6452.8) -> 6432.4 (6437.9) MB, pooled: 0 MB, 977.83 / 0.00 ms  (+ 10.2 ms in 0 steps since start of marking, biggest step 0.0 ms, walltime since start of marking 993 ms) (average mu = 0.24

<--- JS stacktrace --->

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
----- Native stack trace -----

 1: 0x5f639c7edd95 node::OOMErrorHandler(char const*, v8::OOMDetails const&) [next-server (v14.2.15)]
 2: 0x5f639ccc1ba4 v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, v8::OOMDetails const&) [next-server (v14.2.15)]
 3: 0x5f639ccc1f79 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, v8::OOMDetails const&) [next-server (v14.2.15)]
 4: 0x5f639cf0d1bc  [next-server (v14.2.15)]
 5: 0x5f639cf0d1e4  [next-server (v14.2.15)]
 6: 0x5f639cf2453e  [next-server (v14.2.15)]
 7: 0x5f639cf27744  [next-server (v14.2.15)]
 8: 0x5f639d97a667  [next-server (v14.2.15)]

**Comment by xyt-dev** [comment]
This works normally until Node 23.4.0:
```ts
const vivo_sans = localFont({
src: [
{ path: "Font/vivoSans-Regular.ttf", weight: '500' },
{ path: "Font/vivoSans-Bold.ttf", weight: '700' },
],
variable: '--font-vivo-sans',
})
```
Failed to load font file: Font/vivoSans-Bold.ttf
Error: Unknown font format
○ Compiling / ...
The user aborted a request.

Retrying 1/3...
The user aborted a request.

Retrying 1/3...
The user aborted a request.

Retrying 1/3...

<--- Last few GCs --->

[506930:0x5f63b7b4e000] 208231 ms: Scavenge (interleaved) 6435.6 (6444.3) -> 6432.7 (6452.8) MB, pooled: 0 MB, 7.87 / 0.00 ms (average mu = 0.229, current mu = 0.186) allocation failure;
[506930:0x5f63b7b4e000] 209436 ms: Mark-Compact (reduce) 6433.0 (6452.8) -> 6432.4 (6437.9) MB, pooled: 0 MB, 977.83 / 0.00 ms (+ 10.2 ms in 0 steps since start of marking, biggest step 0.0 ms, walltime since start of marking 993 ms) (average mu = 0.24

<--- JS stacktrace --->

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
----- Native stack trace -----

1: 0x5f639c7edd95 node::OOMErrorHandler(char const*, v8::OOMDetails const&) [next-server (v14.2.15)]
2: 0x5f639ccc1ba4 v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, v8::OOMDetails const&) [next-server (v14.2.15)]
3: 0x5f639ccc1f79 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, v8::OOMDetails const&) [next-server (v14.2.15)]
4: 0x5f639cf0d1bc [next-server (v14.2.15)]
5: 0x5f639cf0d1e4 [next-server (v14.2.15)]
6: 0x5f639cf2453e [next-server (v14.2.15)]
7: 0x5f639cf27744 [next-server (v14.2.15)]
8: 0x5f639d97a667 [next-server (v14.2.15)]

**Comment by Shellishack** [comment]
Upgrading to Node v23.5.0 solved this issue for me.

**Comment by milkyfuckinway** [comment]
Fixed in Node v23.5.0
